packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -253,6 +253,14 @@ export class DeltaManager
         return this.lastProcessedSequenceNumber;
     }
 
+    public get clientSequenceNumberObserved(): number {
+        return this._clientSequenceNumberObserved;
+    }
+
+    public get clientSequenceNumber(): number {
```

Comment:
I'd prefer not to expose these properties, and instead expose single boolean that computes this.clientSequenceNumberObserved !== this.clientSequenceNumber